File: DSA/Aditya_Sahni/Day1_Ques1.java
public class que1 {
    public static void main(String[] args) {
        int[] arr = {2, 4, 1, 3};
        int day = 10;
        int month = 4;
        System.out.println(numberOfsegments(arr, day, month));
    }

    static int numberOfsegments(int[] arr, int day, int month) {
        int count = 0;

        for (int i = 0; i <= arr.length - month; i++) {
            int sum = 0;

            for (int j = i; j < i + month; j++) {
                sum += arr[j];
            }
            if (sum == day) {
                count++;
            }
        }
        
        return count;
    }
}

File: DSA/Harshita/Day1_Ques1.cpp
#include<iostream>
using namespace std;
int arrayy(int arr[], int n, int day, int month){
  int count = 0;
  for(int i = 0; i<=n-month; i++){
    int sum = 0;
    for(int j = 0; j<month; j++){
      sum = sum+arr[i+j];
      if(sum == day){
        count++;
      }
    }
  }
  return count;
}
int main(){
  int arr[] = {2,2,1,3,2};
  int day = 4, month = 2;
  int n = sizeof(arr)/sizeof(arr[0]);
  int a = arrayy(arr,n,day,month);
  cout<<a;
}
